libsolidity/formal/CHC.cpp

```diff
@@ -292,6 +331,18 @@ void CHC::endVisit(FunctionCall const& _funCall)
 	createReturnedExpressions(_funCall);
 }
 
+void CHC::endVisit(Break const&)
+{
+	eraseKnowledge();
+	m_context.resetVariables([](VariableDeclaration const&) { return true; });
```

Comment:
Is this enough to erase or do we need to erase all knowledge again after the loop again (as with "m_unknownFunctionCallSeen" in Line 283/284)?